cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/DocCommand.java

```diff
@@ -250,8 +250,8 @@ public void execute() {
             targetPath = this.sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid Ballerina source path, it should either be a module name in a Ballerina project or a " +
-                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension. use the -a or --all " +
+                    "invalid Ballerina source path. It should either be a name of a module in a Ballerina project or " +
+                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension. use the -a or --all " +
```

Comment:
Build seems to use uppercase U for use. Should this be the same?

```suggestion
                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension. Use the -a or --all " +
```